app/models/cms/content_node.rb

```diff
@@ -169,23 +160,8 @@ def correct_redirect
       end
     end
 
-    def redirect_path
```

Comment:
was this  not used or why did you delete it?

Reply:
yeah, I couldn't find a place where this is used and couldn't find any specs that would fail because of it. Probably a dead code